src/city/sentiment.c

```diff
@@ -35,6 +37,22 @@
 #define IMPERIAL_GAMES_SENTIMENT_BONUS 15
 #define POP_STEP_FOR_BASE_AVERAGE_HOUSE_LEVEL 625
 
+#define min(a,b)             \
```

Comment:
Looks like MSVC doesn't support these kind of macros, so MSVC builds fail. Could you perhaps add it as a function instead? Or maybe even just inline it. It looks like it's only being used once.